plugins/services/src/js/structs/Framework.js

```diff
@@ -1,15 +1,54 @@
 import { cleanServiceJSON } from "#SRC/js/utils/CleanJSONUtil";
 import { isSDKService } from "#PLUGINS/services/src/js/utils/ServiceUtil";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
 
 import {
   ROUTE_ACCESS_PREFIX,
   FRAMEWORK_ID_VALID_CHARACTERS
 } from "../constants/FrameworkConstants";
 import FrameworkUtil from "../utils/FrameworkUtil";
+import ServiceStatus from "../constants/ServiceStatus";
 
 import Application from "./Application";
 import FrameworkSpec from "./FrameworkSpec";
 
+const frameworkStatusHelper = (displayName, key, priority) => ({
+  displayName,
+  key,
+  priority
+});
+
+// prettier-ignore
+/* eslint-disable no-multi-spaces */
+const FrameworkStatus = {
+  418: frameworkStatusHelper("Initializing",                    ServiceStatus.DEPLOYING,  1),
+  200: frameworkStatusHelper("Running",                         ServiceStatus.RUNNING,    1),
+  500: frameworkStatusHelper("Error Creating Service",          ServiceStatus.WARNING,    1),
```

Comment:
Should we introduce an `ERROR` status instead of using `WARNING` here? the design has groups with `Error` status and `Warning` status and it seems confusing to use `ServiceStatus.WARNING` when we want the status text to be `Error ...` and the icon to be `CircleClose` instead of `Yield`

Reply:
[ServiceStatus](https://github.com/dcos/dcos-ui/blob/ph/feature/DCOS-49583-Accurate-Status/plugins/services/src/js/constants/ServiceStatus.ts#L20-L53) does not have a `WARNING` status defined at all currently.